app/javascript/dashboard/components/ChatList.vue

```diff
@@ -544,6 +552,11 @@ export default {
       this.$store.dispatch('customViews/update', payloadData);
       this.closeAdvanceFiltersModal();
     },
+    setSavedSortByFilter() {
```

Comment:
The function name can improve, we also update the status, so let's rename it to something like `setFiltersFromUISettings`